react-components/src/util/DateFormatter.js

```diff
@@ -3,17 +3,16 @@
  */
 
 /**
- *
- * @param format
  * @param longDate
+ *  @param format
```

Comment:
Don't forget this JSDoc is still incomplete. I would also move it just above the `function formatDate` definition (below the imports).

I haven't tried adding docs to an `export default function ...` statement, but hopefully it still works 👍